File: Easy/Java/1716. Calculate Money in Leetcode Bank.java
class Solution {
    public int totalMoney(int n) {
        int mon = n / 7; // Get number of full weeks
        int bank = 0;

        // Money from full weeks
        for(int i = 1; i <= mon; i++){
            bank += 7 * (i + 3);
        }

        // Money for leftover days
        for(int i = 7 * mon; i < n; i++){
            mon++;
            bank += mon;
        }
        
        return bank;
    }
}

File: Easy/Python/1716. Calculate Money in Leetcode Bank.py
class Solution(object):
    def totalMoney(self, n):
        mon = n / 7 # Get number of full weeks
        bank = 0

        # Money from full weeks
        for i in range(1, mon + 1, 1):
            bank += 7 * (i + 3)
        
        # Money for leftover days
        for i in range(7 * mon, n, 1):
            mon += 1
            bank += mon
        
        return bank
